### DISCUSSION TITLE
Access form methods from a different component

### DISCUSSION THREAD

**Discussion by j-lee8** [react-hook-form/react-hook-form#6464] [Q&A]
Is this possible?

I have a form but need to add one of its fields into the modal header instead (different components). I'm not sure how this is supposed to work??

I also need the isValid, isSubmitting accessible in the modal header as well for the action buttons.

Thanks all!

**Comment by JamesWong1999** [comment]
https://react-hook-form.com/api/useformcontext
https://react-hook-form.com/api/useformstate

The above hooks should resolve your problem.
